src/robot/model/body.py

```diff
@@ -223,15 +240,18 @@ class Body(BaseBody):
     pass
 
 
-class Branches(BaseBody):
+class Branches(BaseBody, Generic[T]):
```

Comment:
This is wrong. `Try` and `If` objects have `body` that has type `Branches`. `Branches` doesn't contain `Try` and `If` objects, it contains `TryBranch` and `IfBranch` objects.

Also `__init__` below has wrong typing due to this.